### PR TITLE
[KHRGA-29] Updated DA Images

### PR DESCRIPTION
**Pull request by PetroBondar**
Issue: KHRGA-29

### CODE REVIEW

File: source/iface/image.rst [hunk 1]
```diff
@@ -8,341 +8,861 @@
...
-``allocator``
-``pitch``
-``hostPointer``
-``syclContext``
-``clMemObject``
-``availableEvent``
-==================  ===
+================  ==========
+``hostPointer``   Pointer to host memory to hold data.
+``format``        :ref:`image_format` that describe layout of the picture.
```
Comment (mkinsner) [inline source/iface/image.rst:280]:
```suggestion
``format``        :ref:`image_format` that describe layout of the image.
```

File: source/iface/image.rst [hunk 2]
```diff
@@ -8,341 +8,861 @@
...
+will be derived from the ``format`` parameter.
+
 .. rubric:: Parameters
 
-=================  =======
-flag
-=================  =======
+================  ==========
+``hostPointer``   Pointer to host memory to hold data.
+``format``        :ref:`image_format` that describe layout of the picture.
```
Comment (mkinsner) [inline source/iface/image.rst:574]:
```suggestion
``format``        :ref:`image_format` that describe layout of the image.
```

File: source/iface/image.rst [hunk 3]
```diff
@@ -8,341 +8,861 @@
...
-  void set_final_data(Destination finalData = nullptr);
+  void set_final_data(Destination finalData = nullptr)
 
-Description
+The ``finalData`` point to where the output of all the
+image processing is going to be copied to at destruction
+time, if the image was involved with a write accessor.
 
-.. rubric:: Template parameters
+Destination can be either an output iterator, a ``std::weak_ptr<T>``.
```
Comment (mkinsner) [inline source/iface/image.rst:394]:
```suggestion
Destination can be either an output iterator, or a ``std::weak_ptr<T>``.
```

Comment (mkinsner) [inline source/iface/image.rst:394]:
```suggestion
Destination can be either an output iterator, or a ``std::weak_ptr<T>``.
```

**Comment by PetroBondar** [comment]
Fixed all findings in the https://github.com/KhronosGroup/SYCL_Reference/pull/99/commits/4c2399d4cdc515ca8938ada1bf371a9b36e28f9b.

**Review by mkinsner — APPROVED** [review]

**Pull request merged**